<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_department</class>
 <widget class="QDialog" name="Dialog_department">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Departments</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView_department"/>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_deptTitle">
         <property name="text">
          <string>Department title:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_deptTitle"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddDept">
       <property name="text">
        <string>Add department</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChgDept">
       <property name="text">
        <string>Change department</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnDelDept">
         <property name="text">
          <string>Delete department</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
